package.json

```diff
@@ -36,6 +36,9 @@
         "uuid": "^8.3.2"
     },
     "devDependencies": {
+        "@nomicfoundation/hardhat-chai-matchers": "^1.0.0",
```

Comment:
I think these test should be deps, not dev deps, but open to discussion. My definition of dev deps are things that you need if you are WORKING on the examples. Every developer that wants to USE the examples will need these deps, right?

Reply:
If I move hardhat-chai-matchers to dependencies would it no make sense at that point to move chai and hardhat-ethers as well?